packages/schemix/src/schemix/schema.py

```
@@ -0,0 +1,103 @@
+                parts.append(f"DEFAULT {int(column.default_value)}")
+            else:
+                parts.append(f"DEFAULT {str(column.default_value).upper()}")
+        else:
+            parts.append(f"DEFAULT {column.default_value}")
+
+    return " ".join(parts)
+
+
+def generate_foreign_key_constraint_sql(column: ColumnType, dialect: Dialect) -> str:
+    """Generate SQL for foreign key constraint."""
+    if not column._references:
```

Pylint (3.3.7)

[warning] Access to a protected member _references of a client class

(W0212)